llvm/test/Analysis/UniformityAnalysis/AMDGPU/phi_no_divergence.ll

```diff
@@ -0,0 +1,114 @@
+; RUN: opt %s -mtriple amdgcn-- -passes='print<uniformity>' -disable-output 2>&1 | FileCheck %s
+
```

Comment:
Would be useful to have one or two lines explaining what's the distinguishing pattern in each function compared to other functions in the file.